projects/observability/src/shared/components/cartesian/d3/axis/cartesian-axis.ts

```diff
@@ -1,5 +1,6 @@
 import { Axis as D3Axis, axisBottom, AxisDomain, axisLeft, axisRight, AxisScale, axisTop } from 'd3-axis';
 import { BaseType, select, Selection } from 'd3-selection';
+import * as d3 from 'd3-time';
```

Comment:
A '*' import like this would import all modules from d3 and increase the bundle size. We can just do `import {timeDay} from 'd3-time'`

Reply:
Yes, fixed